src/Dashboard/Dashboard.tsx

```diff
@@ -1,32 +1,21 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
-import { getRandomCocktail } from '../apiCalls';
 import './Dashboard.scss';
 import { Cocktail } from '../Definitions/RandomCocktail'
 
 export interface DashboardProps {
-	username: string
+	username: string;
+	randomCocktail: Cocktail
 }
 
 const Dashboard: React.SFC<DashboardProps> = (props) => {
-	const [randomCocktail, setRandomCocktail] = useState<Cocktail>({idDrink: '', strDrink: '', strInstructions: '', strDrinkThumb: ''});
+	const [randomCocktail, setRandomCocktail] = useState<Cocktail>(props.randomCocktail);
```

Comment:
makes more sense!

Reply:
100% cleaner.  I'm glad Alex you got that definitions file in place.